nginx-basicauth/docker-compose.yml

```diff
@@ -23,38 +23,38 @@ services:
       - "${NGINX_PORT:-9000}:${NGINX_PORT:-9000}"
     environment:
       - FORWARD_HOST=redisinsight
-      - FORWARD_PORT=${RIPORT:-8001}
+      - FORWARD_PORT=${RIPORT:-5540}
       - NGINX_PORT=${NGINX_PORT:-9000}
       - BASIC_USERNAME=${BASIC_USERNAME:-redis}
       - BASIC_PASSWORD=${BASIC_PASSWORD:-password}
     command:
       - bash
       - -c
       - |
-        printf "$$BASIC_USERNAME:$$(openssl passwd -crypt $$BASIC_PASSWORD)\n" >> /etc/nginx/.htpasswd
+        printf "$$BASIC_USERNAME:$$(openssl passwd -1 $$BASIC_PASSWORD)\n" >> /etc/nginx/.htpasswd
```

Comment:
this is important to restore auth functionality (-crypt does not work).

Reply:
Correct. `-crypt` is unavailable in the latest OpenSSL.

Reply:
`-1` does the same.

```
Cryptographic options:
 -salt val           Use provided salt
 -6                  SHA512-based password algorithm
 -5                  SHA256-based password algorithm
 -apr1               MD5-based password algorithm, Apache variant
 -1                  MD5-based password algorithm
 -aixmd5             AIX MD5-based password algorithm
```